docs/authentication/saml/jit-provisioning.mdx

```diff
@@ -0,0 +1,17 @@
+---
+description: Learn about Clerk's support for Just-In-Time account provisioning with all supported SAML providers.
+---
+
+# JIT account provisioning with SAML
+
+Just-in-Time (JIT) account provisioning, or automatic account provisioning, is a process by which accounts for employees are created on-demand when they sign-up to an app with SAML using a company email address.
+
+Using JIT account provisioning means your IT department won't have to manually create user accounts for each of the services or apps your employees use to get work done. Consider using this feature if you don't need to control who accesses your Clerk application on a case-by-case basis.
+
+Clerk supports JIT account provisioning for all [supported SAML providers](/docs/pr/1038/authentication/saml/overview) out of the box. This feature is available for any Clerk app that has the **[Sync user attributes during Sign in](/docs/authentication/saml/overview#sync-user-attributes-during-sign-in)** setting enabled, which is enabled by default.
```

Comment:
The Sync user attributes during Sign in setting, only controls the behavior during sign in. We should move the relevant section from the SAML overview page to this page

```suggestion
Clerk supports Just-in-Time (JIT) Provisioning for all [supported SAML providers](/docs/pr/1038/authentication/saml/overview) out of the box.
```